lab_1_keywords_tfidf/main.py

```diff
@@ -118,7 +150,26 @@ def calculate_expected_frequency(
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not(isinstance(doc_freqs, dict) and doc_freqs != {}
+            and all(isinstance(token, str) for token in doc_freqs.keys())
+            and all(isinstance(freq, int) for freq in doc_freqs.values())
+            and isinstance(corpus_freqs, dict)
+            and all(isinstance(token, str) for token in corpus_freqs.keys())
+            and all(isinstance(freq, int) for freq in corpus_freqs.values())):
+        return None
+
+    expected_dict = {}
+    doc_words_sum = sum(doc_freqs.values())
+    collection_words_sum = sum(corpus_freqs.values())
+    for token, freq in doc_freqs.items():
+        if token not in corpus_freqs.keys():
+            corpus_freqs[token] = 0
+
+        expected = (((freq + corpus_freqs[token]) * (freq + doc_words_sum - freq)) /
```

Comment:
`freq + doc_words_sum - freq` virtually equals to just `doc_words_sum`, is this the right value according to formula?